db/migrateWpPostsToArchieMl.ts

```diff
@@ -92,7 +92,8 @@ const migrate = async (): Promise<void> => {
             )
 
             const archieMlFieldContent: OwidArticleType = {
-                id: `wp-${post.id}`,
+                id: 1,
```

Comment:
is this correct?

Reply:
I looked yesterday, and I _think_ that this id is never used anywhere.